interface/forms/functional_cognitive_status/new.php

```diff
@@ -142,97 +127,97 @@ function checkVal(id)
-                                    <input type="hidden" id="codetext_1" name="codetext[]" class="codetext" value="<?php echo text($obj["codetext"]); ?>">
-                                </div>
-                                <div class="forms col-5">
-                                    <label for="description_1" class="h5"><?php echo xlt('Description'); ?>:</label>
-                                    <textarea name="description[]"  id="description_1" class="form-control description"  rows="3" ><?php echo text($obj["description"]); ?></textarea>
-                                </div>
-                                <div class="forms col-2">
-                                    <label for="code_date_1" class="h5"><?php echo xlt('Date'); ?>:</label>
-                                    <input type='text' id="code_date_1"  name='code_date[]' class="form-control code_date datepicker" value='<?php echo attr($obj["date"]); ?>' title='<?php echo xla('yyyy-mm-dd Date of service'); ?>' />
-                                </div>
-                                <div class="forms col-1">
-                                    <label for="activity_1" class="h5"><?php echo xlt('Active'); ?>:&nbsp;</label>
-                                    <br />
-                                    <input type="checkbox" name="activity[]" onclick="checkVal(this.id);" id="activity_1" value="0" class="activity">
-                                    <input type="hidden" name="activity1[]" id="activity1_1" value="0" class="activity1">
-                                </div>
-                                <div class="forms col-1" style="padding-top: 35px">
-                                    <i class="fa fa-plus-circle fa-2x" aria-hidden="true" onclick="duplicateRow(this.parentElement.parentElement.parentElement);" title='<?php echo xla('Click here to duplicate the row'); ?>'></i>
-                                    <i class="fa fa-times-circle fa-2x text-danger"  aria-hidden="true" onclick="deleteRow(this.parentElement.parentElement.parentElement.id);"  title='<?php echo xla('Click here to delete the row'); ?>'></i>
+                            <?php }
+                            ?>
+                        </fieldset>
+                        <div class="form-group clearfix">
+                            <div class="col-sm-12 position-override">
+                                <div class="btn-group oe-opt-btn-group-pinch" role="group">
+                                    <button type="submit" onclick="top.restoreSession()" class="btn btn-primary btn-save"><?php echo xlt('Save'); ?></button>
+                                    <button type="button" class="btn btn-secondary btn-cancel oe-opt-btn-separate-left" onclick="top.restoreSession(); parent.closeTab(window.name, false)"><?php echo xlt('Cancel');?></button>
+                                    <input type="hidden" id="clickId" value="" />
```

Comment:
Move hidden input outside of btn-group as the btn-group gets messed up due to this